kitty/project.yaml

```diff
@@ -8,7 +8,29 @@ network:
   endpoint: "ws://host.docker.internal:9944"
   types: {
     "KittyIndex": "u32",
-    "Kitty": "[u8; 16]"
+    "Kitty": "[u8; 16]",
+  }
+  typesBundle: {
+    spec: {
+      chainname: {
+        rpc: {
+          kitties: {
+            getKittyPrice:{
+                description: string;
+                params: [
+                    {
+                      name: 'kittyIndex',
+                      type: 'KittyIndex',
+                      isHistoric: true,
```

Comment:
How a paramater of type 'KittyIndex' to be a historic parameter? Ususally you find a parameter of type BlockHash